packages/connectivity/lib/connectivity.dart

```diff
@@ -12,7 +12,28 @@ import 'package:meta/meta.dart';
 /// WiFi: Device connected via Wi-Fi
 /// Mobile: Device connected to cellular network
 /// None: Device not connected to any network
-enum ConnectivityResult { wifi, mobile, none }
+enum ConnectionType { wifi, mobile, none }
+enum ConnectionSubtype { EDGE, HSDPA, LTE, none, unknown }
+
+class ConnectivityResult {
```

Comment:
Seems to be a breaking change introducing this class.

Reply:
Reverted these changes and implemented a new API